docs/specs/validation.yml

```diff
@@ -88,3 +88,200 @@ outputs:
     ["warning","field-required","Missing required field 'a'","a.md",2,1]
     ["warning","unexpected-type","Expect type 'String' but got 'Array'","a.md",2,6]
 ---
+# Json schema microsoft alias rule: The author is in the allow list
+inputs:
+  docfx.yml: |
+    metadataSchema:
+      - schema.json
+  schema.json: |
+    { 
+      "properties":
+      { 
+        "ms.author":
+        { 
+          "microsoftAlias": 
+          {
+            "allowedDLs":["amlstudiodocs", "o365devx"]
+          }
+        }
+      } 
+    }
+  a.md: |
+    ---
+    ms.author: o365devx
+    ---
+outputs:
+  a.json: |
+    { "ms.author": "o365devx" }
+---
+# Json schema microsoft alias rule: The author is not in the allow list and without graph call
+inputs:
+  docfx.yml: |
+    metadataSchema:
+      - schema.json
+  schema.json: |
+    { 
+      "properties":
+      { 
+        "ms.author":
+        { 
+          "microsoftAlias": 
+          {
+            "allowedDLs":["amlstudiodocs", "o365devx"]
+          }
+        }
+      } 
+    }
+  a.md: |
+    ---
+    ms.author: over123
+    ---
+outputs:
+  a.json:
+  .errors.log: |
+    ["warning","ms-alias-invalid","Invalid value for 'ms.author', 'over123' is not a valid Microsoft alias","a.md",2,12]
```

Comment:
does this mean if not token is applied, we always assume everything is not a valid ms alias? This may not be the desired local behavior. Because in local experience, we never have access token, show warning in this case would produce a huge amount of errors, suggest to __NOT__ show error when token is not there.

Reply:
You are right, I have already designed those logic